lib/srv/discovery/database_watcher.go

```diff
@@ -116,7 +117,24 @@ func (s *Server) onDatabaseCreate(ctx context.Context, resource types.ResourceWi
 	if trace.IsAlreadyExists(err) {
 		return trace.Wrap(s.onDatabaseUpdate(ctx, resource))
 	}
-	return trace.Wrap(err)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	err = s.emitUsageEvents(map[string]*usageeventsv1.ResourceCreateEvent{
+		"db/" + database.GetName(): {
```

Comment:
nit: What about placing the prefix into a constant? ([similar to what we do on presence](https://github.com/gravitational/teleport/blob/master/lib/services/local/presence.go#L1846)).